### PR TITLE
fix: dismissing reactions after 5s

### PR DESCRIPTION
**Pull request by Brazol**

### CODE REVIEW

File: docusaurus/docs/Flutter/03-core-concepts/11-reactions.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Reactions
+slug: /reactions
+description: How to use reactions during the call
+---
+
+## Reactions
+​
+It's easy to add reactions to your call.
```
Comment (deven98) [inline docusaurus/docs/Flutter/03-core-concepts/11-reactions.mdx:9]:
```suggestion
Reactions in video calling enable participants to express emotions and non-verbal cues without interrupting the flow of conversation. It also helps create a more interactive and responsive virtual meeting environment.

It is easy to create reactions with Stream Video's Flutter SDK.
```

File: docusaurus/docs/Flutter/03-core-concepts/11-reactions.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+```dart
+await call.sendReaction(reactionType: 'fireworks');
+```
+
+You can also add custom data to the reaction and specify a specific emoji
```
Comment (deven98) [inline docusaurus/docs/Flutter/03-core-concepts/11-reactions.mdx:15]:
```suggestion
You can also add custom data to the reaction and specify a specific emoji:
```

File: packages/stream_video/lib/src/call/call.dart [hunk 3]
```diff
@@ -58,6 +58,7 @@ const _tag = 'SV:Call';
 
 const _reconnectTimeout = Duration(seconds: 30);
 const _fastReconnectTimeout = Duration(seconds: kDebugMode ? 10 : 3);
+const _reactionAutoDismissTime = Duration(seconds: 5);
```
Comment (deven98) [inline packages/stream_video/lib/src/call/call.dart:61]: Should we somehow make this customisable?

Comment (Brazol) [reply packages/stream_video/lib/src/call/call.dart]: Added 👍

**Review by deven98 — APPROVED** [review]
LGTM

**Pull request merged**